SecurityPkg/Library/Tcg2PreUefiEventLogLibNull/Tcg2PreUefiEventLogLibNull.c

```diff
@@ -0,0 +1,20 @@
+/** @file Tcg2PreUefiEventLogLibNull.c
+NOTE: NULL Tcg2PreUefiEventLogLibNull library instance
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Library/DebugLib.h>
+
+/**
+  Create the EventLog entries.
+**/
+VOID
+CreateTcg2PreUefiEventLogEntries (

```

Comment:
Need `EFIAPI` for externally facing interfaces.

```suggestion
VOID
EFIAPI
CreateTcg2PreUefiEventLogEntries (

```

Reply:
added EFIAPI